lib/generators/active_admin/resource/templates/admin.rb

```diff
@@ -1,5 +1,5 @@
 ActiveAdmin.register <%= class_name %> do
-<% if Rails::VERSION::MAJOR == 4 || defined?(ActionController::StrongParameters) %>
+<% if Rails::VERSION::MAJOR >= 4 || defined?(ActionController::StrongParameters) %>
```

Comment:
:+1: 

I think this code was originally added before we had the dependency adapter.